packages/web/src/components/spa/HackAUBG/JudgesSection/JudgesSection.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import axios from 'axios';
+import { useState } from 'react';
+import { useEffect } from 'react';
+import { url, openNewTab } from '../../../../Global';
+import { JudgesCard } from './JudgesCard';
+import './judges_section.css';
+
+export const JudgesSection = () => {
+    const [jury, setJury] = useState([{}]);
+
+    const getJury = () => {
+        axios({
+            method: 'get',
+            url: url + '/api/jury'
+        })
+            .then((res) => {
+                setJury(res.data.data.data);
+            })
+            .catch((err) => {
+                console.log(err);
+            });
+    };
+
+    const renderJudges = () => {
+        if (jury) {
+            return (
+                <div className="judges-container">
+                    <h1 className="judges-header">Judges</h1>
+                    <div className="judges-picture">
+                        {jury.map((judge, index) => (
+                            <div key={index} className="judge-div">
+                                <JudgesCard
+                                    Judge={judge}
```

Comment:
judge

Reply:
Elaborate please where and what exactly I have to change. If you are talking about the passed parameter judge in line 34, its already the same word/spelling.

Reply:
Currently the prop name is Judge instead of judge